src/test/java/unbanner/SectionTest.java

```diff
@@ -172,4 +174,22 @@ public void addSectionToRoomListTest() {
     section.addSectionToRoomList(room);
     assertTrue(room.sectionList.contains(section));
   }
+
+  @Test
+  public void setSemesterTest() {
```

Comment:
this test will have to be removed, we can write a new one once Harley finishes with the semester class.